src/fu-tool.c

```diff
@@ -1685,6 +1731,12 @@ main (int argc, char *argv[])
 		     /* TRANSLATORS: command description */
 		     _("Update the stored metadata with current contents"),
 		     fu_util_verify_update);
+	fu_util_cmd_array_add (cmd_array,
+		     "verify-blob",
```

Comment:
As there is no verification happening by the engine how do you feel about dump blob instead? Or read firmware or read blob?

Reply:
I did long term envisage wiring this up to be able to verify the hardware by storing the blob somewhere on the filesystem rather than just the checksum in a database. Not sure if that's crazy. `firmware-dump` was the other command name I toyed with.

Reply:
Or since we already have install-blob and get-*, maybe get-blob?

Reply:
the `get` commands at the moment just get the internal state without progress. Maybe we want `read-firmware`?

Reply:
I think that longer term vision should be a separate command though from this rather than morphing later. Keep one as dump and another later that combines dump and verify.

Reply:
Since we just added something called `firmware-parse`, how about either `firmware-dump` or `firmware-read` then?